terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/validation/bytemin.jsp

```diff
@@ -0,0 +1,18 @@
+<h2>@ByteMin Test</h2>

```

Comment:
@ikeyat 
 Sorry, It's difference that may occur due to I had to rebase too much.
 I Canceled the wrong rebase, and re-push only the necessary commits.
 This difference does not occur.